cmake/doc.cmake

```diff
@@ -6,6 +6,13 @@ endif()
 
 string(TIMESTAMP TODAY "%B %d, %Y")
 
+# Ensure the doc directory exists for pandoc versions that won't create it.
+add_custom_target(
+    CreateDocDir ALL
+    COMMAND ${CMAKE_COMMAND} -E make_directory ${CMAKE_CURRENT_BINARY_DIR}/doc
+)
+add_dependencies(CreateDocDir CreateDocDir)
```

Comment:
can you check this line, i'm not sure what it should be.

you could also move `COMMAND ${CMAKE_COMMAND} -E make_directory ${CMAKE_CURRENT_BINARY_DIR}/doc` to the `add_custom_command` below, before invoking pandoc.

Reply:
oh I didn't realize you could add multiple commands, MUCH better, thanks!